exporters/prometheus/testdata/empty_resource.txt

```diff
@@ -0,0 +1,3 @@
+# HELP foo a simple counter
+# TYPE foo counter
+foo{A="B",C="D",E="true",F="42"} 24.3
```

Comment:
@dashpole should we still export a `target_info` metric for an empty resource or can we just skip it? Couldn't find out what is the expected behavior for this case on the spec

Reply:
great question.  I think it should still be included.  That is because the outcome of the promql query:

`my_metric * on (job, instance) group_left() target_info`

Changes depending on whether target_info is included or not.  If it is included with no labels, you will still get my_metric in the output (unchanged).  If it is excluded, my_metric will be excluded from the output.  If I am running multiple applications, some of which have resource information, and some of which don't, having the empty target_info is useful as it means all endpoints have the target_info metric.